config/development.json

```diff
@@ -159,7 +159,7 @@
 		"press-this": true,
 		"publicize-preview": true,
 		"purchases/new-payment-methods": true,
-		"purchases/purchase-list-dataview": true,
+		"purchases/purchase-list-dataview": false,
```

Comment:
@sirbrillig I'm disabling the feature flag in development as it's making E2E tests fail locally.